yeonjae/programmers-0/컨트롤제트.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+  const arr = s.split(" ");
+  const result = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] === "Z") {
+      result.pop();
+    } else {
+      result.push(+arr[i]);
+    }
+  }
+
+  return result.reduce((a, b) => {
+    return a + b;
+  }, 0);
+}
```

Comment:
```suggestion
function solution(s) {
  const arr = s.split(" ");
  const result = [];

  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === "Z") {
      result.pop();
    } else {
      result.push(+arr[i]);
    }
  }

  return result.reduce((a, b) => a + b, 0);
}
```
- 잘푸셨습니다 :) reduce에서 중괄호 제거하고 그냥 a + b만 적으셔도 좋을 것 같아요 👍